templates/ts/m365-message-extension/teamsapp.yml.tpl

```diff
@@ -48,7 +48,7 @@ provision:
           # variable before ARM deployment.
           parameters: ./infra/azure.parameters.json
           # Required when deploying ARM template
-          deploymentName: Create-resources-for-tab
+          deploymentName: Create-resources-for-mxt
```

Comment:
```suggestion
          deploymentName: Create-resources-for-me
```

Reply:
Or `-for-me-search` to keep consistent with other me samples.

Reply:
I think it's a bit confusing to abbreviate "message extension" to "me". Others may not know what "me" stand for.
@MuyangAmigo @summzhan What are your opinions on this?

Reply:
I see message-extension uses `-mxt` and message-extension-action uses `-me-action`...

Reply:
@hund030 will use template maintenance tools to unify these later.